src/Todo/Todo.module.css

```diff
@@ -0,0 +1,3 @@
+.title {
+  color: #3f51b5;;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/dotodo/pullRequest?prid=5932591)